test/sql/aggregate/aggregates/test_distinct_ungrouped.test

```diff
@@ -0,0 +1,191 @@
+# name: test/sql/aggregate/aggregates/test_distinct_ungrouped.test
+# description: DISTINCT aggregations, without GROUP BY
+# group: [aggregates]
+
+# Since these tests are made to test the ungrouped operator, and not necessarily the functions themselves
+# This test will mostly focus on the order and mixing of distinct and non-distinct aggregates
+# And not on variation between types and functions
+
+#distinct aggregate =	'D'
+#regular aggregate =	'-'
+
+statement ok
+PRAGMA enable_verification
+
+statement ok
+create table tbl as
```

Comment:
* Can we add a `.test_coverage` file that runs an ungrouped count distinct with many rows and with `PRAGMA verify_parallelism`? Both with many groups (e.g. 1M+ groups) and with a few groups (10 groups) so we can test both the partitioned and the unpartitioned case. Also a test where we mix both a small distinct (unpartitioned) and a large distinct (partitioned) would be great.
* Can we add a test with an ungrouped min(distinct) with lists and long strings, to verify data ownership is handled correctly?

Reply:
I've added the tests, though I wonder if I interpreted your instructions correctly

Reply:
By groups, you are referring to groups in the hashtable right? So basically how many distinct values the dataset contains
(i%10 from range(10000)) -> 10 groups ?

Reply:
For the second point, I've created a table with lists of strings, that each have one element, and are pretty big
```sql
statement ok
create or replace table tbl as select * FROM ( VALUES
	([(select string_agg(i, '') from (select 'a' as i from range(1000000) tbl(i)) as tbl)]),
	([(select string_agg(i, '') from (select 'a' as i from range(1000000) tbl(i)) as tbl)]),
	([(select string_agg(i, '') from (select 'a' as i from range(1000000) tbl(i)) as tbl)]),
	([(select string_agg(i, '') from (select 'a' as i from range(1000000) tbl(i)) as tbl)]),
	([(select string_agg(i, '') from (select 'b' as i from range(1000000) tbl(i)) as tbl)]),
	([(select string_agg(i, '') from (select 'b' as i from range(1000000) tbl(i)) as tbl)]),
	([(select string_agg(i, '') from (select 'b' as i from range(1000000) tbl(i)) as tbl)]),
	([(select string_agg(i, '') from (select 'b' as i from range(1000000) tbl(i)) as tbl)]),
) tbl(i)

query I
select (min(distinct i)::TEXT)[2:2] from tbl;
----
a
```

Reply:
Yes, both sound good to me